### PR TITLE
Update ProPhyle version

### PR DESCRIPTION
**Pull request by karel-brinda**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/prophyle/meta.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ package:
   #sha256:
 
 build:
-  number: 0
+  number: 1
```
Comment (druvus) [inline recipes/prophyle/meta.yaml]: Build number should restart at 0 for new versions.

File: recipes/prophyle/meta.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ package:
...
   skip: True # [py27]
   script: pip install --upgrade git+https://github.com/karel-brinda/prophyle
```
Comment (bgruening) [inline recipes/prophyle/meta.yaml]: Can we please change this recipe to use the `source` section and not github. Thanks.

Comment (karel-brinda) [inline recipes/prophyle/meta.yaml]: @bgruening Hi Björn, thank you for your comment. ProPhyle is still in a testing phase and does not have any Github releases, yet. This is why the source section is currently not used. Once a stable version is ready, we plan to create a standard Github release and use the source section of the receipt properly.

However, we might use the source section already now – with the PyPI releases (https://pypi.python.org/pypi?:action=display&name=prophyle).  Do you know how to derive a PyPI package address from the version (to be able to update the receipt just by incrementing a single Jinja variable)?

Comment (bgruening) [inline recipes/prophyle/meta.yaml]: I'm not sure I understood the question, but this URL you can use: https://pypi.python.org/packages/bc/59/d5bf13ee886b86ecaaf8e86cd6cf98adc1cf65709de02130f61ae78d2041/prophyle-0.1.0.34.tar.gz
md5=29d6ee0a1c41694342464c31b3826140

Comment (karel-brinda) [inline recipes/prophyle/meta.yaml]: This is the url I was looking for:
https://pypi.io/packages/source/p/prophyle/prophyle-0.1.0.34.tar.gz
It does not contain any hashes.

Comment (bgruening) [inline recipes/prophyle/meta.yaml]: This one is even better. For your information, you can also use github with a URL similar to this:https://github.com/karel-brinda/prophyle/archive/3a4e3ab0be3e10ad5f15b69bce0749cb812c360d.tar.gz The hash is your latest commit.

Comment (karel-brinda) [inline recipes/prophyle/meta.yaml]: Wow, I did not now this. Thank you!

**Comment by bgruening** [comment]
```
RuntimeError: Setuptools downloading is disabled in conda build. Be sure to add all dependencies in the meta.yaml  url=https://pypi.python.org/simple/psutil/r
```
psutil is missing as dependency

**Pull request merged**